test-cases/cdc/cdc-15m-replication.yaml

```diff
@@ -0,0 +1,38 @@
+test_duration: 3600
```

Comment:
Many of the params here aren't really needed as they are part of the defaults or will be overriden as part of the pipeline file.
I'll help you to clean it after you have a pipeline file.

Reply:
yes need to clean the unneeded params that are part of defaults/defined in Jenkins pipeline